mgr.go

```diff
@@ -261,20 +264,18 @@ func newSysboxMgr(ctx *cli.Context) (*SysboxMgr, error) {
 		logrus.Info("Sys container DNS aliasing disabled.")
 	}
 
-	if !mgrCfg.useShiftfs {
-		logrus.Info("Shiftfs disabled or not supported.")
+	if ctx.GlobalBool("disable-shiftfs") {
```

Comment:
I placed them there because I wanted to group all the print-outs, as opposed to having them a bit more scattered (see all existing print-outs below the ones I added).